DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/textSearch/highlightQueryResultsBasics/metadata.json

```diff
@@ -0,0 +1,235 @@
+                "  * `termVector()`<br>",
+                "     Set _FieldTermVector.WITH_POSITIONS_AND_OFFSETS_ to store the term's position and offsets.",
+                "",
+                "* During the indexing process, the field's content is tokenized, broken into distinct terms according to the configured analyzer (RavenStandardAnalyzer is used by default).",
+                "",
+                "* A query can now be made on these terms to get their matching documents - including a list of text fragments containing the highlighted searched terms.",
+                "  See next step."
+            ],
+            "Assets": [
+                {
+                    "Title": "FTS - Static Index Demo",
+                    "Type" : "demo",
+                    "Url"  : "/java/text-search/fts-with-static-index-single-field"
+                },
+                {
+                    "Title": "Store Fields on Index Demo",
+                    "Type" : "demo",
+                    "Url"  : "/java/static-indexes/store-fields-on-index"
+                },
+                {
+                    "Title": "Querying highlighting",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/latest/java/indexes/querying/highlighting"
+                },
+                {
+                    "Title": "Indexes - Analyzers",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/latest/java/indexes/using-analyzers"
+                },
+                {
+                    "Title": "Using term vectors",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/latest/java/indexes/using-term-vectors"
+                },
+                {
+                    "Title": "Term vector in Lucene",
+                    "Type" : "link",
+                    "Url"  : "http://makble.com/what-is-term-vector-in-lucene"
+                }
+            ]
+        },
+        {
+            "Title": "Make a search query",
+            "Slug" : "step-4",
+            "DescriptionLines": [
+                "* Query the 'Employees' collection using the defined index for documents that contain the specified term.",
+                "",
+                "* In this example,<br> we are searching for documents containing the term 'sales' within their 'Notes' field.<br>",
+                "  In addition, we request the highlighted fragments.",
+                "",
+                "* `highlight()`<br>",
+                "   * Specify the index field for which to get text fragments with the term highlighted.<br>",
+                "   * fragmentLength - the maximum length for a fragment (length must be >= 18).<br>",
+                "   * fragmentCount - used to limit the number of matching fragments returned per document.<br>",
+                "   * Provide an 'out Highlightings' param for the fragments' results.",
```

Comment:
No `out` in Java 
So just change to:
=>
Provide a 'Highlightings' param for the fragments' results

Reply:
Done